tensorboard/__init__.py

```diff
@@ -20,7 +20,17 @@
 
 from tensorboard import errors  # public export
 from tensorboard import lazy
-from tensorboard import version
+from tensorboard import version  # public export
+
+# TensorBoard public API.
+__all__ = [
+    "__version__",
+    "errors",
+    "notebook",
+    "program",
+    "summary",
+    "version",
```

Comment:
There's no particular reason to make this public, right?  Since we have `__version__` which is what https://www.python.org/dev/peps/pep-0396/ recommends.  So even though it could be public IMO simpler overall to skip it.

Reply:
Sure, fine with me. (I thought that it was meant to be public since
`__version__` didn’t always exist, but I don’t feel strongly about it.)